docs/features/collection.adoc

```diff
@@ -106,10 +106,12 @@ Following is a list of output tuning options based upon the https://github.com/o
 |Delivery
 a|The mode for log forwarding.
 
-- AtLeastOnce (default): The forwarder will block in an attempt to deliver all messages.  When the tuning spec is added to an output, this additionally configures an internal, durable buffer so the collector can attempt to forward any logs read before it restarted
+- AtLeastOnce: The forwarder will block in an attempt to deliver all messages.  When the tuning spec is added to an output, this additionally configures an internal, durable buffer so the collector can attempt to forward any logs read before it restarted
```

Comment:
@vparfonov please review the API comments as there may be something similar that also needs to be addressed